### PR TITLE
test(cypress): add test Admin/Chronograf/All Org pages

### PR DESCRIPTION
**Pull request by k3yi0**
This PR adds;

- adds All Organization test
- adds data-test attributes

### CODE REVIEW

File: ui/src/shared/components/InputClickToEdit.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ class InputClickToEdit extends PureComponent<Props, State> {
         </div>
       </div>
     ) : (
-      <div className={wrapperClass}>
+      <div className={wrapperClass} data-test="rename-org">
```
Comment (sranka) [inline ui/src/shared/components/InputClickToEdit.tsx:120]: "rename-org" id is not used in tests, is it?

Comment (k3yi0) [reply ui/src/shared/components/InputClickToEdit.tsx:120]: No it's not, should I remove it?

Comment (sranka) [reply ui/src/shared/components/InputClickToEdit.tsx:120]: Yes, pls and yes, a shared component must not rely upon instance-specific ID

File: ui/src/shared/components/InputClickToEdit.tsx [hunk 2]
```diff
@@ -130,13 +132,15 @@ class InputClickToEdit extends PureComponent<Props, State> {
             onFocus={this.handleFocus}
             tabIndex={tabIndex}
             spellCheck={false}
+            data-test={`${testId}--input`}
```
Comment (sranka) [inline ui/src/shared/components/InputClickToEdit.tsx:135]: nit: this generates `undefined--input` on unspecified testId ... it should be better be undefined

**Review by sranka — APPROVED** [review]
LGTM

**Pull request merged**